Python/Product/Analysis/Intellisense/ImportedModuleNameWalker.cs

```diff
@@ -36,7 +36,7 @@ class ImportedModuleNameWalker : PythonWalkerWithLocation {
         public IEnumerable<string> ImportedModules { get; private set; } = Enumerable.Empty<string>();
 
         public override bool Walk(FromImportStatement node) {
-            if (node.StartIndex <= Location && Location <= node.EndIndex) {
+            if (node.Root != null && node.Root.StartIndex <= Location && Location <= node.Root.EndIndex) {
```

Comment:
@MikhailArkhipov I just suggested this change to you on your PR. If you take it there as well, it should merge just fine and I'm pretty sure it doesn't affect the other changes you made either.